pkg/issuer/acme/dns/route53/route53.go

```diff
@@ -81,6 +82,7 @@ func (d *sessionProvider) GetSession() (*session.Session, error) {
 		sessionOpts.Config.Credentials = credentials.NewStaticCredentials(d.AccessKeyID, d.SecretAccessKey, "")
 		// also disable 'ambient' region sources
 		sessionOpts.SharedConfigState = session.SharedConfigDisable
+		sessionOpts.Config.HTTPClient = http.DefaultClient
```

Comment:
Oops, that's something that isn't actually needed; I added that because I needed aws-sdk-go to work with my proxy, and I didn't realize that a nil http.Client meant that it would be using http.DefaultClient.

tl;dr: aws-sdk-go already honors the proxy settings by using http.DefaultClient